### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by eapenzacharias**
Build a To-Do List JavaScript application

- Set up a project with webpack
- Created an index.html file and added HTML markup with empty placeholders
- Created an index.js file and set an array of some sample to-do task data
- Created a function to iterate over the tasks array and populate the HTML
- On page load, the dynamically created list of tasks is rendered into the dedicated placeholder
- Created a style.css from style.scss and set rules for the To-Do List and loaded the CSS using Webpack CSS loader

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "todolist-v2",
+  "version": "1.0.0",
+  "description": "> Build a To Do List JavaScript application",
+  "private": true,
+  "scripts": {
```
Comment (rloterh) [inline package.json:6]: - [ ] The _script option_ of your _package.json_ file is **not properly configured** and hence your application does not launch in a browser when the command `npm start` is run. Kindly append this line below to the _script options:_ 
`"start": "webpack serve --open"`, in your _packge.json_ file.

**Your script option in package.json file:**
image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "url": "git+https://github.com/eapenzacharias/todolist-v2.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/eapenzacharias/todolist-v2/issues"
+  },
+  "homepage": "https://github.com/eapenzacharias/todolist-v2#readme",
+  "devDependencies": {
```
Comment (rloterh) [inline package.json:22]: - [ ] The _dervServer_ package is missing your application. Kindly install the same and add that in your **webpack** configuration, by running the command  `npm install -D webpack-dev-server`
to enable your application to be run on the server locally.

**devServer in webpack configuration file:**
image

File: src/scripts/printInterface.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { getElement, createElement } from './queries.js';
+
+function subMenu() {
```
Comment (rloterh) [inline src/scripts/printInterface.js:3]: - [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi , 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @eapenzacharias , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**